core/vm/instructions.go

```diff
@@ -911,6 +912,21 @@ func opETX(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte
 		return nil, nil
 	}
 
+	if etxGasLimit.CmpUint64(math.MaxUint64) == 1 {
```

Comment:
should this be == 1? or >= 1?

Reply:
CmpUint64 returns 1 if etxGasLimit is greater than math.MaxUint64